tests/features/common/page-objects/jobs-and-workflows.po.js

```diff
@@ -452,7 +452,7 @@ module.exports = {
         'div.create-container div.create-container__data div.accordion__container.functions-wrapper h5.data-header__title span'
       ),
       Collapse_Button: By.css(
-        'div.create-container div.create-container__data div.accordion__container.functions-wrapper button.accordion__icon'
+        'div.create-container div.create-container__data div.accordion__container.functions-wrapper div.accordion__icon'
```

Comment:
```suggestion
        '.create-container .create-container__data .accordion__container.functions-wrapper .accordion__icon'
```
I think such a selector presentation will be better.